pkgs/development/python-modules/dash/default.nix

```diff
@@ -47,6 +49,8 @@ buildPythonPackage rec {
       tests/unit/dash/
   '';
 
+  doCheck = true;
+
```

Comment:
That is the default. Also pytest should be run in checkPhase, use pytestCheckHook instead.

Reply:
Thanks for the quick review @SuperSandro2000 !

Hopefully, commit https://github.com/NixOS/nixpkgs/pull/139778/commits/69ba6afac1e9ebf4a6525e0b5649573b60fa1200 addresses those concerns.